bzt/jmx.py

```diff
@@ -264,7 +264,8 @@ def _get_arguments_panel(name):
                              guiclass="ArgumentsPanel", testclass="Arguments")
 
     @staticmethod
-    def _get_http_request(url, label, method, timeout, body, keepalive, files=(), encoding=None):
+    def _get_http_request(url, label, method, timeout, body, keepalive, files=(), encoding=None,
+                          redirect_follow=True, redirect_auto=False):
```

Comment:
Do we really need two options? Aren't they same thing and nobody understand the difference?